### PR TITLE
WPF/WindowsBase:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool.

### PR DESCRIPTION
**Pull request by ryalanms**
WPF/WindowsBase:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool.

The RC1 source for these assemblies has 'triple-slash' comments not in the documentation:

*PresentationBuildTasks
PresentationCore
ReachFramework
System.Windows.Controls.Ribbon
System.Windows.Presentation
UIAutomationClient
UIAutomationTypes
WindowsBase*

There is one documentation update PR per assembly. 

(Hide whitespace changes when reviewing files.) 

/cc @dotnet/wpf-developers @carlossanlop

### CODE REVIEW

File: xml/System.Windows/VectorConverter.xml [hunk 1]
```diff
@@ -1,203 +1,207 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ To determine whether a conversion is possible without actually performing the conversion, use the <xref:System.Windows.VectorConverter.CanConvertTo%2A> method.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.NotSupportedException">A NotSupportedException is thrown if "value" is <see langword="null" /> or not an instance of Vector,
+            or if the destinationType isn't one of the valid destination types.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows/VectorConverter.xml:203]:
```suggestion
        <exception cref="T:System.NotSupportedException"><paramref name="value"/> is <see langword="null" />.

-or-

<paramref name="value"/> cannot be converted to a <see cref="T:System.Windows.Vector"/>.

-or-

<paramref name="destinationType"/> is not one of the valid destination types.</exception>

```

File: xml/System.Windows/VectorConverter.xml [hunk 2]
```diff
@@ -1,203 +1,207 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ To determine whether a conversion is possible without actually performing the conversion, use the <xref:System.Windows.VectorConverter.CanConvertFrom%2A> method.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.NotSupportedException">A NotSupportedException is thrown if the example object is <see langword="null" /> or is not a valid type
+            which can be converted to a Vector.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows/VectorConverter.xml:160]: I think this is referring to the value, right?
```suggestion
        <exception cref="T:System.NotSupportedException"><paramref name="value"/> is <see langword="null" />.

-or-

<paramref name="value"/> cannot be converted to a <see cref="T:System.Windows.Vector"/>.</exception>

```

File: xml/System.Windows/SizeConverter.xml [hunk 3]
```diff
@@ -1,193 +1,195 @@
...
+      <Docs>
+        <param name="context">Provides contextual information about a component.</param>
+        <param name="culture">Culture-specific information that should be respected during conversion.</param>
+        <param name="value">The source object that is being converted.</param>
+        <summary>Attempts to convert a specified object to an instance of <see cref="T:System.Windows.Size" />.</summary>
+        <returns>The instance of <see cref="T:System.Windows.Size" /> that is created from the converted <paramref name="source" />.</returns>
+        <remarks>To be added.</remarks>
+        <altmember cref="T:System.Windows.Size" />
+        <exception cref="T:System.NotSupportedException">A NotSupportedException is thrown if the example object is <see langword="null" /> or is not a valid type
+            which can be converted to a Size.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows/SizeConverter.xml:155]: I think "example object" is referring to the value, right?

```suggestion
        <exception cref="T:System.NotSupportedException"><paramref name="value"/> is <see langword="null" />.

-or-
<paramref name="value"/> cannot be converted to a <see cref="T:System.Windows.Size"/>.</exception>

```

**Review by carlossanlop — COMMENTED** [review]
Left some suggestions for the exceptions.

File: xml/System.Windows/Freezable.xml [hunk 4]
```diff
@@ -1,1101 +1,1103 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ If you attempt to add or remove event handlers when the current object is not modifiable (when the <xref:System.Windows.Freezable.IsFrozen%2A> property is `true`), an exception is thrown.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">An attempt was made to modify the Changed handler of
+            a value with IsFrozen == <see langword="true" />.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows/Freezable.xml:149]:
```suggestion
        <exception cref="T:System.InvalidOperationException">An attempt was made to modify the Changed handler of a value with IsFrozen == <see langword="true" />.</exception>

```

File: xml/System.Windows/DependencyObject.xml [hunk 5]
```diff
@@ -1,782 +1,791 @@
...
+      <Parameters />
+      <Docs>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.DependencyObject" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Automatic determination of current Dispatcher. Use alternative constructor
+                that accepts a Dispatcher for best performance.
```
Comment (carlossanlop) [inline xml/System.Windows/DependencyObject.xml:87]: Is this remark relevant for the final user, or is it directed to code maintainers?

File: xml/System.Windows.Threading/DispatcherPriorityAwaiter.xml [hunk 6]
```diff
@@ -1,104 +1,112 @@
...
+    </Interface>
+  </Interfaces>
+  <Docs>
+    <summary>Represents an object that waits for the completion of an asynchronous task.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is returned from DispatcherPriorityAwaitable.GetAwaiter()
```
Comment (carlossanlop) [inline xml/System.Windows.Threading/DispatcherPriorityAwaiter.xml:29]: Is this relevant to the final user?

File: xml/System.Windows.Threading/DispatcherOperation.xml [hunk 7]
```diff
@@ -1,420 +1,438 @@
...
+      <Docs>
+        <summary>Occurs when the operation has completed.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Completed indicates that the operation was invoked and has
+                either completed successfully or faulted. Note that a canceled
+                or aborted operation is never is never considered completed.
```
Comment (carlossanlop) [inline xml/System.Windows.Threading/DispatcherOperation.xml:128]:
```suggestion
`Completed` indicates that the operation was invoked and has either completed successfully or faulted. Note that a canceled or aborted operation is never considered completed.

```

File: xml/System.Windows.Threading/DispatcherHooks.xml [hunk 8]
```diff
@@ -1,271 +1,285 @@
...
+
+## Remarks
+
+Typically this is due to the BeginInvoke API, but the Invoke API can
+                also cause this if any priority other than DispatcherPriority.Send is
+                specified, or if the destination dispatcher is owned by a different
+                thread.
+                <P />
+                Note that any thread can post operations, so this event can be
+                raised by any thread.
```
Comment (carlossanlop) [inline xml/System.Windows.Threading/DispatcherHooks.xml:278]:
```suggestion
Typically this is due to the `BeginInvoke` API, but the `Invoke` API can also cause this if any priority other than `DispatcherPriority.Send` is specified, or if the destination dispatcher is owned by a different thread.

Note that any thread can post operations, so this event can be raised by any thread.

```

File: xml/System.Windows.Threading/Dispatcher.xml [hunk 9]
```diff
@@ -1,2490 +1,2506 @@
...
+          <Attributes>
+            <Attribute FrameworkAlternate="net-5.0">
+              <AttributeName Language="C#">[System.ParamArray]</AttributeName>
+              <AttributeName Language="F#">[&lt;System.ParamArray&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+      </Parameters>
+      <Docs>
+        <param name="priority">The priority, relative to the other pending operations in the <see cref="T:Syste
```
Comment (carlossanlop) [inline xml/System.Windows.Threading/Dispatcher.xml:1820]:
```suggestion
Note that the default priority is `DispatcherPriority.Normal`.

```

Comment (carlossanlop) [inline xml/System.Windows.Threading/Dispatcher.xml:922]:
```suggestion
Note that the default priority is `DispatcherPriority.Send`.

```

File: xml/System.Windows.Data/DataSourceProvider.xml [hunk 10]
```diff
@@ -1,688 +1,721 @@
...
+      <Docs>
+        <summary>Gets or sets the current <see cref="T:System.Windows.Threading.Dispatcher" /> object to the UI thread to use.</summary>
+        <value>The current <see cref="T:System.Windows.Threading.Dispatcher" /> object to the UI thread to use. By default, this is the <see cref="T:System.Windows.Threading.Dispatcher" /> object that is associated with the thread on which this instance was created.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+By default, this is the Dispatcher associated with the thread
+            on which this DataProvider instance was created.
```
Comment (carlossanlop) [inline xml/System.Windows.Data/DataSourceProvider.xml:255]:
```suggestion
By default, this is the `Dispatcher` associated with the thread on which this `DataProvider` instance was created.

```

File: xml/System.Windows.Data/DataSourceProvider.xml [hunk 11]
```diff
@@ -1,688 +1,721 @@
...
+      </ReturnValue>
+      <Docs>
+        <summary>Gets or sets the current <see cref="T:System.Windows.Threading.Dispatcher" /> object to the UI thread to use.</summary>
+        <value>The current <see cref="T:System.Windows.Threading.Dispatcher" /> object to the UI thread to use. By default, this is the <see cref="T:System.Windows.Threading.Dispatcher" /> object that is associated with the thread on which this instance was created.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+By default, this is the `Dispatcher` associated with the thread on which this `DataProvider` instance was created.
```
Comment (gewarren) [inline xml/System.Windows.Data/DataSourceProvider.xml:254]: This is already stated in the `<value>` comments.

File: xml/System.Windows.Threading/Dispatcher.xml [hunk 12]
```diff
@@ -1,2490 +1,2507 @@
...
```
Comment (gewarren) [inline xml/System.Windows.Threading/Dispatcher.xml:922]:
```suggestion
The default priority is `DispatcherPriority.Send`.
```

Comment (gewarren) [inline xml/System.Windows.Threading/Dispatcher.xml:1820]:
```suggestion
The default priority is `DispatcherPriority.Normal`.
```

File: xml/System.Windows.Threading/DispatcherHooks.xml [hunk 13]
```diff
@@ -1,271 +1,282 @@
...
+      <Docs>
+        <summary>Occurs when an operation is invoked.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Typically this is due to the `BeginInvoke` API, but the `Invoke` API can also cause this if any priority other than `DispatcherPriority.Send` is specified, or if the destination dispatcher is owned by a different thread.
+
+Note that any thread can post operations, so this event can be raised by any thread.
```
Comment (gewarren) [inline xml/System.Windows.Threading/DispatcherHooks.xml:274]:
```suggestion
Any thread can post operations, so this event can be raised by any thread.
```

File: xml/System.Windows.Threading/DispatcherHooks.xml [hunk 14]
```diff
@@ -1,271 +1,282 @@
...
+        <ReturnType>System.Windows.Threading.DispatcherHookEventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when an operation is invoked.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Typically this is due to the `BeginInvoke` API, but the `Invoke` API can also cause this if any priority other than `DispatcherPriority.Send` is specified, or if the destination dispatcher is owned by a different thread.
```
Comment (gewarren) [inline xml/System.Windows.Threading/DispatcherHooks.xml:272]:
```suggestion
Typically, this event is due to the `BeginInvoke` API, but the `Invoke` API can also cause this event if any priority other than `DispatcherPriority.Send` is specified, or if the destination dispatcher is owned by a different thread.

```

File: xml/System.Windows.Threading/DispatcherOperation.xml [hunk 15]
```diff
@@ -1,420 +1,437 @@
...
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the operation has completed.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+`Completed` indicates that the operation was invoked and has either completed successfully or faulted. Note that a canceled or aborted operation is never considered completed.
```
Comment (gewarren) [inline xml/System.Windows.Threading/DispatcherOperation.xml:126]:
```suggestion
`Completed` indicates that the operation was invoked and has either completed successfully or faulted. A canceled or aborted operation is never considered completed.
```

File: xml/System.Windows/Freezable.xml [hunk 16]
```diff
@@ -1,1101 +1,1103 @@
...
+        <summary>Occurs when the <see cref="T:System.Windows.Freezable" /> or an object it contains is modified.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ If you attempt to add or remove event handlers when the current object is not modifiable (when the <xref:System.Windows.Freezable.IsFrozen%2A> property is `true`), an exception is thrown.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">An attempt was made to modify the Changed handler of a value with IsFrozen == <see langword="true" />.</exception>
```
Comment (gewarren) [inline xml/System.Windows/Freezable.xml:148]:
```suggestion
        <exception cref="T:System.InvalidOperationException">An attempt was made to modify the <code>Changed</code> handler of a value with <code>IsFrozen == true</code>.</exception>
```

File: xml/System.Windows/VectorConverter.xml [hunk 17]
```diff
@@ -1,203 +1,215 @@
...
+        <returns>The object created from converting this <see cref="T:System.Windows.Vector" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ To determine whether a conversion is possible without actually performing the conversion, use the <xref:System.Windows.VectorConverter.CanConvertTo%2A> method.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.NotSupportedException"><paramref name="value"/> is <see langword="null" />.
```
Comment (gewarren) [inline xml/System.Windows/VectorConverter.xml:202]:
```suggestion
        <exception cref="T:System.NotSupportedException">
          <paramref name="value"/> is <see langword="null" />.

```

**Review by gewarren — APPROVED** [review]
